web/src/conversation_participants.ts

```diff
@@ -0,0 +1,33 @@
+import type {Message} from "./message_store.ts";
```

Comment:
Let's add a comment at the top explaining this data structure.

Reply:
done.